### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by visualcookie** [fintory/boot-next#5]
## Motivation

Right now to document the structure and motivation behind it, the `README.md` files are in seperate folders. To emit having a bloated `README.md` file in our projects, we should move further documentation to the Wiki.

## Tasks

- [ ] Setup structure for Wiki pages
- [ ] Add content

**Comment by visualcookie** [comment]
# Structure

## TOC
- COMING SOON ...

**Comment by iDuuck** [comment]
@visualcookie I already set up a basic structure, please see the branch for documentation.

**Comment by visualcookie** [comment]
> @visualcookie I already set up a basic structure, please see the branch for documentation.

Yeah, I was not done with the TOC yet. I wanted to basically ad the TOC to this issue, so we know what we have to add to the docs.
